Google.Cloud.EntityFrameworkCore.Spanner/Extensions/MethodInfoExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Reflection;
```

Comment:
nit: All files should have a copyright header with the current year (so 2022 in this case)